module-service/src/main/java/module/service/ShowingServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package module.service;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+import dto.internal.MovieDTO;
+import dto.internal.ShowingDTO;
+import lombok.RequiredArgsConstructor;
+import module.entity.Showing;
+import module.repository.ShowingRepository;
+
+@Service
+@RequiredArgsConstructor
+public class ShowingServiceImpl implements ShowingService {
+
+	private final ShowingRepository showingRepository;
+	private final ModelMapper modelMapper;
+
+	@Override
+	public List<Map.Entry<MovieDTO, List<ShowingDTO>>> getTodayShowing() {
```

Comment:
```suggestion
	public Map<MovieDTO, List<ShowingDTO>> getTodayShowing() {
```
이렇게 하면 같은거 아닌가요? 좀 더 간결해보이네요.
MovieDTO의 개봉일 정렬은 적용하지 못하겠네요

Reply:
하지만!!!
근본적인 것으로 돌아가보면 MovieDTO와 ShowingDTO를 나눠야 하는 이유가 있나요?
두 개를 포함하는 DTO를 만들어보는건 어때요?